src/utils/add-corpus-storage-accounts/requirements.txt

```diff
@@ -1,3 +1,3 @@
 azure-mgmt-storage~=18.0.0
-azure-cli-core==2.19.1
```

Comment:
@bmc-msft  Do I need to do any other testing besides integration test ? 
Any reason this azure-cli-core was pinned to way older version compare to the other ones ?

Reply:
I don't believe there was a reason that it was pinned to previous versions.

The integration tests do make use of this utility.  You should test this tool on it's own.